
# Failover

In the case of unexpected errors on the primary for longer than the .spec.failoverDelay (by default 0 seconds), the cluster will go into failover mode. This may happen, for example, when:

- The primary pod has a disk failure
- The primary pod is deleted
- The postgres container on the primary has any kind of sustained failure.

In the failover scenario, the primary cannot be assumed to be working properly.

# Command
```
kubectl delete \\
  pvc/${primary} \\
  pod/${primary} \\
  pvc/${primary}-tbs-idx \\
  pvc/${primary}-tbs-tmptbs \\
  pvc/${primary}-wal \\
  --force
```

# Doc
[Failover](https://cloudnative-pg.io/documentation/1.25/failover/)
